### PR TITLE
D8CORE-2470: Add process plugin to check image dimensions

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Add process plugin that will skip if the image is too small

# Need Review By (Date)
- 10/15

# Urgency
- medium

# Steps to Test
1. see steps in https://github.com/SU-SWS/stanford_profile/pull/308

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: tests/src/Unit/Plugin/migrate/process/ImageDimensionSkipTest.php [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      'width' => 100,
+      'method' => 'process',
+    ], '', []);
+    $migrate = $this->createMock(MigrateExecutableInterface::class);
+    $row = $this->createMock(Row::class);
+
+    $value = 'http://placecorgi.com/260/180';
+    $this->assertEquals($value, $plugin->transform($value, $migrate, $row, ''));
+
+    $value = 'http://placecorgi.com/50/50';
```
Comment (sherakama) [inline tests/src/Unit/Plugin/migrate/process/ImageDimensionSkipTest.php:82]: Nice choice.

**Review by sherakama — APPROVED** [review]
So this is the sneaky way of avoiding the dreaded "black pixel"?

**Comment by pookmish** [comment]
yep. Right now, my profile would be imported with a 1x1 black dot without this work.

**Comment by pookmish** [comment]
I learned some about the cap image permissions yesterday so if you're interested, I can fill you in.

**Pull request merged**